### PR TITLE
Board: Add create / edit card label popup

### PR DESCRIPTION
**Pull request by annano**
Signed-off-by: Anna No [anna.no@xored.com](mailto:anna.no@xored.com)

### CODE REVIEW

File: plugins/board-resources/src/components/presenters/ColorPresenter.svelte [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+</script>
+
+{#if value}
+  <div
+    class="border-radius-1 min-w-9"
+    class:h-8={size === 'large'}
+    class:h-7={size === 'medium'}
+    class:h-6={size === 'small'}
+    style:background-color={isHovered ? numberToRGB(value, 0.6) : numberToHexColor(value)}
```
Comment (haiodo) [inline plugins/board-resources/src/components/presenters/ColorPresenter.svelte]: why not use :hover css style?

File: plugins/board-resources/src/components/presenters/LabelPresenter.svelte [hunk 2]
```diff
@@ -1,18 +1,14 @@
 <script lang="ts">
-  import { Button, numberToHexColor } from '@anticrm/ui'
   import type { CardLabel } from '@anticrm/board'
+import ColorPresenter from './ColorPresenter.svelte'
```
Comment (haiodo) [inline plugins/board-resources/src/components/presenters/LabelPresenter.svelte]: indentation

**Review by haiodo — APPROVED** [review]

File: plugins/board-resources/src/components/presenters/ColorPresenter.svelte (hunk 1 above)
Comment (annano) [reply plugins/board-resources/src/components/presenters/ColorPresenter.svelte]: Updated

File: plugins/board-resources/src/components/presenters/LabelPresenter.svelte (hunk 2 above)
Comment (annano) [reply plugins/board-resources/src/components/presenters/LabelPresenter.svelte]: Updated

**Pull request merged**